CMakeLists.txt

```diff
@@ -1,37 +1,32 @@
-#: # CMakeLists.txt ([source](../appskeleton/CMakeLists.txt))
-#: 
-#: `CMakeLists.txt` contains the build configuration that can be used to build
-#: your App without any IDE or to generate project files for your specific IDE.
-#: 
-#: ## Required settings for all apps
-cmake_minimum_required(VERSION 3.5)
-
-#: This sets the minimum required CMake version.
-#: Here, [CMake 3.5](https://cmake.org/cmake/help/latest/release/3.5.html#modules)
-#: was chosen because it is the first version to support imported targets
-#: (e.g. `boost::thread`). Before setting this to a higher version, please check
-#: if it has arrived in [debian stable](https://packages.debian.org/stable/cmake).
-project(BestPracticesGUI
+# CMakeLists.txt contains the build configuration that can be used to build
+# your App without any IDE or to generate project files for your specific IDE.
+# https://packages.ubuntu.com/bionic/cmake
```

Comment:
This is a bit strange. Why link to the ubuntu package here?

Reply:
The main linux distributions are Debian and Ubuntu derivates so the required CMake version shouldn't be newer than the one on most developers' PCs.

Reply:
The links below make sense. The single link above this comment is the strange one.

Reply:
Right, also added to the fixup queue